transport_nantes/topicblog/templates/topicblog/panel_did_you_know_tip_1_mail_client.html

```diff
@@ -1,5 +1,4 @@
 {% load markdown %}
-<meta name="viewport" content="width=device-width,initial-scale=1.0">
```

Comment:
I wasn't even sure about what it was about, maybe the responsive aspect of the mail ? The media queries will need the width, so maybe it's used to get proper values regarding the width to perform responsive, but that's just a guess really